cocos/2d/assets/bitmap-font.ts

```diff
@@ -102,33 +102,39 @@ export class FontAtlas {
  */
 @ccclass('cc.BitmapFont')
 export class BitmapFont extends Font {
+    /**
+     * @deprecated since v3.7.0, Useless Code.
```

Comment:
Are you sure it's not working, or do you recommend not using it?

Reply:
It's not working